dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,8 +18,8 @@
 role.administrator=Administrator Role
 role.administrator.desc=A collection of users, groups, or both that are assigned the server administrator role.
 
-role.viewer=Viewer Role
-role.viewer.desc=A collection of users, groups, or both that are assigned the server viewer role.
+role.reader=Reader Role
+role.reader.desc=A collection of users, groups, or both that are assigned the server reader role.
```

Comment:
All good on the change from Viewer to Reader- but- should this description include what what the Reader role entails? (eg. capable of viewing but editing) Is Reader a standard role with standard permissions that could be succinctly described? As is, this description doesn't describe the role, just restates what's implied by the title

Reply:
Yeah. I questioned this myself. 

Maybe we could do something like:

> reader.desc=A collection of users, groups, or both that are assigned the server reader role. The reader role bestows read-only privileges.

Reply:
Could update the the administrator description in the same fashion, but saying 'bestows reader and write privileges.'

Reply:
Maybe some ideas from here?

https://www.ibm.com/support/knowledgecenter/ro/SSAW57_9.0.0/com.ibm.websphere.nd.multiplatform.doc/ae/rsec_adminroles.html

Reply:
I think that is an improvement- what about "grants" instead of "bestows"?

Also, "server reader role" is a bit hard to parse at a glance. Would "...Reader Role for the server" also be correct?

I notice that, across KC, Reader Role is capitalized. I can see why it might not be in this description, but I think it could help with legibility and consistency to capitalize it, unless there is a specific reason not to.

Reply:
Does this work?

role.reader=Reader Role
role.reader.desc=A collection of users, groups, or both that are granted the Reader role.

Reply:
That works- but I think you include permission description you had above, so:

"role.reader=Reader Role
role.reader.desc=A collection of users, groups, or both that are assigned the Reader role. The reader role grants read-only privileges."

Reply:
I have updated both the reader and administrator role descriptions accordingly.

Reply:
David approved the changes.